crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -3144,7 +3144,7 @@ async fn test_sui_system_state_nop_upgrade() {
 
     let authority_state = init_state().await;
 
-    let protocol_config = ProtocolConfig::get_for_version(ProtocolVersion::MIN);
+    let protocol_config = ProtocolConfig::get_for_version(ProtocolVersion::MAX);
```

Comment:
this may be a question for @lxfind maybe? 
bottom line this test fails if it is not pointing to latest version. I assume that is expected but feedback welcome

Reply:
LGTM